src/core/namespace.c

```diff
@@ -1341,14 +1341,59 @@ static int mount_mqueuefs(const MountEntry *m) {
         return 1;
 }
 
+static int verities_find(const char *path, VeritySettings **verities, size_t *n_verities, VeritySettings **ret_verity) {
```

Comment:
verities_find() sounds as if it was a passive thing that checks and exits. but it does more, it actually appends. maybe call this "verities_look_for_duplicates()"?